service/src/main/java/bio/terra/pipelines/app/configuration/internal/PipelinesCommonConfiguration.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.pipelines.app.configuration.internal;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "pipelines.common")
+public record PipelinesCommonConfiguration(Long storageBucketTtlDays) {}
```

Comment:
so we already have `WdlPipelinesConfiguration`, which contains stuff about quotas. Now that I think about it, quotas won't necessarily be specific to WdlPipelines - I like `PipelinesCommonConfiguration` better for that too. what do you think about merging the quota config stuff into this one?

Reply:
oh good point i totally forgot there was a WdlPipelinesConfiguration.  let me see what each of them have and think about it a bit